src/config.json

```diff
@@ -16,7 +16,7 @@
     "makerfoundation.com",
     "fulcrum.trade",
     "launchpad.ethereum.org"
-    
+
```

Comment:
Nit: May as well get rid of this newline as well, since we're getting rid of that errant whitespace.

```suggestion
```